src/components/Avatar/styled/container.js

```diff
@@ -26,7 +18,7 @@ const StyledContainer = styled.span.attrs(props => {
 
     &:hover,
     &:focus {
-        color: ${props => props.brandContrastTextColor};
+        color: ${props => props.palette.getContrastText(props.palette.brand.main)};
```

Comment:
in situations like this, I think we should maintain the ```styled.span.attrs(props =>{})``` in order to compute getContrastText only once